lib/datawarehouse/index.ts

```diff
@@ -47,7 +49,10 @@ export class ServerlessDataWarehouse extends pulumi.ComponentResource {
         if (this.tables[name]) {
             throw new Error(`Duplicate table! Name: ${name}`);
         }
-        const table = this.createTable(name, args.columns, args.partitionKeys);
+
+        const dataFormat = args.dataFormat ? args.dataFormat : "parquet";
```

Comment:
Should there be additional checking that the dataFormat is acceptable?

Reply:
That union type you commented on down below enforces type safety at compile time. Java script users could potentially be bitten by this though.

Reply:
Added some validation: https://github.com/EvanBoyle/pulumi-serverless-db/pull/14/commits/96319d39634701e8a10afef0bc62d8ebbc898b87